### ISSUE TITLE
Doesn't show errors in files if flow strip_root setting is on

### ISSUE THREAD

**Issue by AgentME** [AtomLinter/linter-flow#65]
If your .flowconfig file has `strip_root=true` in the [options] section, then linter-flow will only show squiggly lines under errors if the file has any unsaved changes.

(I'm not really clear what that setting is good for. Doesn't seem to make any difference in the output of manually running `flow`. Just found someone had turned it on in a project and I was confused as to why linter-flow was acting weird.)

**Comment by nmn** [comment]
related #78

**Comment by nmn** [comment]
Can you check what you get when you run `flow status`?

**Comment by Arcanemagus** [comment]
Closing due to a lack of response. Please respond if you are still having issues in v5.5.0.

**Comment by AgentME** [comment]
The bug still exists in v5.5.0. Adding `strip_root=true` to the [options] of a fresh or existing project reliably triggers it for me. Does it not for anyone else? I've attached a video in case I didn't describe it right: [bug.zip](https://github.com/AtomLinter/linter-flow/files/410115/bug.zip).
`flow` and `flow status` don't have their output affected by the option.

test.js
```js
/* @flow */

var x = y;
```

.flowconfig
```
[ignore]

[include]

[libs]

[options]
strip_root=true
```

Before adding the option
```
$ flow
test.js:3
  3: var x = y;
             ^ identifier `y`. Could not resolve name


Found 1 error
```

After adding the option
```

$ flow
test.js:3
  3: var x = y;
             ^ identifier `y`. Could not resolve name


Found 1 error
$ flow status

test.js:3
  3: var x = y;
             ^ identifier `y`. Could not resolve name


Found 1 error
```
